demo/demo_2DAcoIsoDenQ_DEO2.jl

```diff
@@ -4,10 +4,10 @@ using Devito
 
 configuration!("log-level", "DEBUG")
 configuration!("language", "openmp")
-configuration!("mpi", false)
+# configuration!("mpi", false)
 
-x = SpaceDimension(name="x", spacing=Constant(name="h_x", value=5.0))
-z = SpaceDimension(name="z", spacing=Constant(name="h_z", value=5.0))
+x = SpaceDimension(name="x", spacing=Spacing(name="h_x", is_const=true))
+z = SpaceDimension(name="z", spacing=Spacing(name="h_z", is_const=true))
```

Comment:
Using Constants is not recommended and will lead to issues, `Spacing` is the type designed for this